README.md

```diff
@@ -13,3 +13,23 @@ Prerequisite: [Node.js(LTS version will be fine) and npm](https://docs.npmjs.com
 4. When you in the angular-UI folder, run `npm install`
 5. To build and serves the app locally, run `ng serve`. If it's build successfully, there should be a link that look like this: http://localhost:4200/. 
 6. Navigate to the link in your browser, and your should be able to see the app.
+
+## Node (embedded) setup Guide
+Prerequisites:
+ - [STM32CubeMx](https://www.st.com/en/development-tools/stm32cubemx.html)
+ - Windows Users will need [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+ - The following packages (should be able to install from package manager):
+   - cmake
```

Comment:
specify minimum required version

Reply:
3.15.3 should be fine - We can do 3.13 for flight computer or whatever the default raspbian CMake version is

Reply:
Sounds good